Adjust wording around free / paid / pro / enterprise plans in a few places (#1377)
wandb/docs: content/guides/models/track/limits.md

The `wandb.log` calls in your script utilize a metrics logging API to log your training data to W&B. This API is engaged through either online or [offline syncing]({{< relref "/ref/cli/wandb-sync.md" >}}). In either case, it imposes a rate limit quota limit in a rolling time window. This includes limits on total request size and request rate, where latter refers to the number of requests in a time duration.

> W&B applies rate limits per W&B project. So if you have 3 projects in a team, each project has its own rate limit quota. Users on [Paid plans](https://wandb.ai/site/pricing) have higher rate limits than those on the Free plan.

Reviewer: @mdlinville how about: 

> Users on Paid plans have higher rate limits than those on Unpaid plans.

this then covers the case of unpaid personal entities (which are deprecated but still exist)

> W&B applies rate limits per W&B project. So if you have 3 projects in a team, each project has its own rate limit quota. Users on [Paid plans](https://wandb.ai/site/pricing) have higher rate limits than those on the Free plan.

Author: ```suggestion
W&B applies rate limits per W&B project. So if you have 3 projects in a team, each project has its own rate limit quota. Users on [Paid plans](https://wandb.ai/site/pricing) have higher rate limits than those on an unpaid plan.
```
